tools/PowerShell/Microsoft.WinGet.Client/src/Library/Enable-WinGetLocalManifest.ps1

```diff
@@ -0,0 +1,41 @@
+Function Enable-WinGetLocalManifest
```

Comment:
From @JohnMcPMS on earlier PR:

>These should probably be something like `Set-WinGetAdminSetting -Setting LocalManifest -Value true` rather than creating a new cmdlet for every setting.

From @denelon 
>The PowerShell cmdlet structure tends to use Enable and Disable. It makes them more discoverable. Maybe someone from the community can weigh in on this one. @doctordns what do you think?

From @JohnMcPMS 
>I just expect the possibility of non-boolean values in the future, but it could also be `Enable`/`Disable` for booleans and `Set` in the future for non-booleans.  But the larger takeaway is to make the actual setting that is being modified be a parameter rather than making new cmdlets per setting.

Reply:
As noted somewhere else - the enable verb is used to move a resource to be available or active. Disable does the opposite. I would prefer Set, especially since the target could possibly take non boolean values. 

IMHO `Enable-Adminsetting -Setting` is not great PowerShell. The noun should be the specific thing we are enableling (eg Enable-LocalUser, Enable-PSRemoting).  Enabler-LocalManifest would be better than this. 

What other commands are there that have the noun "LocalManifest" ?

Reply:
I guess I would agree about the noun being specific; However, I still believe that enable and disable are appropriate based on the Microsoft Documentation
```
Disable (d) | Configures a resource to an unavailable or inactive state. For example, the Disable-PSBreakpoint cmdlet makes a breakpoint inactive. This verb is paired with Enable.

Enable (e) | Configures a resource to an available or active state. For example, the Enable-PSBreakpoint cmdlet makes a breakpoint active. This verb is paired with Disable.
```
In this case, it is making the setting active or inactive. A setting can be considered a resource just as much as a breakpoint can be considered a resource